src/marqo/core/embed/embed.py

```diff
@@ -0,0 +1,123 @@
+from timeit import default_timer as timer
+from typing import List, Optional, Union, Iterable, Sequence, Dict, Any, Tuple
+import numpy as np
+
+import marqo.core.unstructured_vespa_index.common as unstructured_common
+from marqo import marqo_docs
+from marqo import exceptions as base_exceptions
+from marqo.api import exceptions as api_exceptions
+from marqo.core import exceptions as core_exceptions
+# We depend on _httprequests.py for now, but this may be replaced in the future, as
```

Comment:
?

Reply:
I'll remove this. It was a leftover from a different module.